### PR TITLE
Picture save game fixes + variable fix

### PR DESCRIPTION
**Pull request by fmatthew5876**
Fix: #2226 
Fix: #2225

### CODE REVIEW

**Comment by fmatthew5876** [comment]
jenkins: test this please

**Comment by fmatthew5876** [comment]
jenkins: test this please

File: src/game_interpreter.cpp [hunk 1]
```diff
@@ -957,15 +957,11 @@ bool Game_Interpreter::CommandControlVariables(RPG::EventCommand const& com) { /
 			break;
 		case 1:
 			// Var A ops B
-			if (com.parameters[0] != 1) {
-				value = Main_Data::game_variables->Get(com.parameters[5]);
-			}
+			value = Main_Data::game_variables->Get(com.parameters[5]);
```
Comment (carstene1ns) [inline src/game_interpreter.cpp:960]: Always interesting to see things getting back to their former state.

Comment (fmatthew5876) [reply src/game_interpreter.cpp:960]: This was a silly optimization gone wrong.

Comment (Ghabry) [reply src/game_interpreter.cpp:960]: This is why we need a test suite. Would have catched this. Maybe, is an edge case. 🤔

**Comment by fmatthew5876** [comment]
jenkins test this please

File: src/game_interpreter.cpp (hunk 1 above)
Comment (fmatthew5876) [reply src/game_interpreter.cpp:960]: Even if the test suite didn't catch it, once the bug is reported we make a new test and it never happens again no matter how many ways we try to refactor or optimize the underlying code.
